proto/api/v1/central_events_service.proto

```diff
@@ -0,0 +1,143 @@
+syntax = "proto3";
+
+option go_package = "v1";
+
+option java_package = "io.stackrox.proto.api.v1";
+
+import weak "google/api/annotations.proto";
+import "api/v1/common.proto";
+import "api/v1/pagination.proto";
+import "google/protobuf/timestamp.proto";
+
+package v1;
+
+// CentralEvents are emitted by Central. They are used to create transparency for users
+// for asynchronous, background tasks. events are part of Central's system health view.
+message CentralEvent {
+    // UUID of the event.
+    string id                                  = 1;
+
+    // Type of the event.
+    CentralEventType type                      = 2;
+
+    // Level associated with the event. The level is categorized into danger, warn, info,
+    // success.
+    CentralEventLevel level                    = 3;
+
+    // Message associated with the event. The message may include detailed information
+    // for this particular event.
+    string message                             = 4;
+
+    // Hint associated with the event. The hint may include different information based
+    // on the type of event. It can include instructions to resolve a event, or
+    // informational hints.
+    string hint                                = 5;
+
+    // Domain associated with the event. A event's domain outlines the feature domain where
+    // the event was created from. As an example, this might be "Image Scanning".
+    // In case of events that cannot be tied to a specific domain, this will be "General".
+    string domain                              = 6;
+
+    // Resource type associated with the event. A event may refer to an underlying resource
+    // such as a particular image. In that case, the resource type will be filled here.
+    string resource_type                       = 7;
+
+    // Resource ID associated with the event. If a event refers to an underlying resource,
+    // the resource ID identifies the underlying resource.
```

Comment:
Based on [the convo here](https://redhat-internal.slack.com/archives/C05GPEER2DV/p1694007062123959), let's also mention that this _sometimes_ refers to the unique ID of the object, but isn't guaranteed to do so (e.g. it may be that this isn't a image SHA but rather its name).